cli/dcnios-cli.py

```diff
@@ -26,116 +26,131 @@
 import env
 import argparse
 
-types = ["dCache", "OSCAR", "S3", "SQS", "generic", "Kafka"]
-typesSSL = ["Kafka"]
-
-def doType(type, function):
-    if type in data["nifi"]:
-        for singularoftype in data["nifi"][type]:
-            function(singularoftype["name"])
-            print(str(function.__qualname__) + " " + singularoftype["name"])
-
-
-def makeActionWithAllType(allType, function):
-    for type in allType:
-        doType(type, function)
 
 def functionAll(function):
     for dic in env.info:
-        if dic["id"] in data["nifi"]:
-            for information in data["nifi"][dic["id"]]:
-                function(information["name"])
-                print(str(function.__qualname__) + " " + information["name"])
-                if "alterations" in information:
-                    for alter in information["alterations"]:
-                        function(alteration.nameActionReturn(alter["action"],information["name"]))
-                        print(str(function.__qualname__) + " " + alteration.nameActionReturn(alter["action"],information["name"]))
+        if dic["id"] in data[env.NIFI_TAG][dic["type"]]:
+            for information in data[env.NIFI_TAG][dic["type"]][dic["id"]]:
+                function(information[env.NAME_TAG])
+                print(str(function.__qualname__) + " " + information[env.NAME_TAG])
+                if env.ALTERATION_ALTERATIONS_TAG in information:
+                    for alter in information[env.ALTERATION_ALTERATIONS_TAG]:
+                        function(alteration.nameActionReturn(alter[env.ALTERATION_ACTION_TAG], information[env.NAME_TAG]))
+                        print(str(function.__qualname__) + " " + alteration.nameActionReturn(alter[env.ALTERATION_ACTION_TAG], information[env.NAME_TAG]))
+    if env.GENERIC_TAG in data[env.NIFI_TAG]:
+        for information in data[env.NIFI_TAG][env.GENERIC_TAG]:
+                function(information[env.NAME_TAG])
+                print(str(function.__qualname__) + " " + information[env.NAME_TAG])
+                if env.ALTERATION_ALTERATIONS_TAG in information:
+                    for alter in information[env.ALTERATION_ALTERATIONS_TAG]:
+                        function(alteration.nameActionReturn(alter[env.ALTERATION_ACTION_TAG], information[env.NAME_TAG]))
+                        print(str(function.__qualname__) + " " + alteration.nameActionReturn(alter[env.ALTERATION_ACTION_TAG], information[env.NAME_TAG]))
+
 
 def deleteAll(nifi):
```

Comment:
Again, three nested `for`. Please, create a ticket to try to improve this part of the code.